share/dialog.sh

```diff
@@ -161,14 +161,19 @@ END
 dialog_prompt()
 {
     $(opt_parse \
-        "+instructions=1                                   | Include instructions on how to navigate dialog window."   \
```

Comment:
I think that having these instructions displayed by default isn't necessary as we're always disabling this. Caller can still enable it if they want